### PR TITLE
improve killsnoop output

### PR DESCRIPTION
**Pull request by makelinux**
##### Checklist

- [x] User-visible and non-trivial changes updated in `CHANGELOG.md`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -61,11 +61,13 @@ and this project adheres to
 - Use `strftime` instead of `elapsed` in `threadsnoop.bt`
   - [#2917](https://github.com/iovisor/bpftrace/pull/2917)
 - Increase PID field width and align to the right in `threadsnoop.bt`
-  - [#2927](https://github.com/iovisor/bpftrace/pull/2927)
+  - [#2928](https://github.com/iovisor/bpftrace/pull/2928)
```
Comment (viktormalik) [inline CHANGELOG.md:64]: Let's put this as a separate PR.

File: CHANGELOG.md [hunk 2]
```diff
@@ -61,11 +61,13 @@ and this project adheres to
...
 - Update runqlen.bt to remove `runnable_weight` field from cfs_rq struct.
   - [#2790](https://github.com/iovisor/bpftrace/pull/2790)
 - Update mdflush.bt to use blkdev.h instead of genhd.h for non-BTF builds.
   - [#2849](https://github.com/iovisor/bpftrace/pull/2849)
+- Add milliseconds to timestamp and align numbers to the right in `threadsnoop.bt`
```
Comment (viktormalik) [inline CHANGELOG.md:69]: typo?
```suggestion
- Add milliseconds to timestamp and align numbers to the right in `killsnoop.bt`
```

File: tools/killsnoop.bt [hunk 3]
```diff
@@ -31,9 +31,9 @@ tracepoint:syscalls:sys_enter_kill
 tracepoint:syscalls:sys_exit_kill
 /@tpid[tid]/
 {
-	time("%H:%M:%S  ");
-	printf("%-6d %-16s %-4d %-6d %d\n", pid, comm, @tsig[tid], @tpid[tid],
-	    args.ret);
+	printf("%-15s %7d %-16s %4d %6d %6d\n",
+		strftime("%H:%M:%S.%f", nsecs),
```
Comment (viktormalik) [inline tools/killsnoop.bt:35]: Are the nanoseconds really necessary? Do you have use-cases where the current granularity is not sufficient?

**Review by viktormalik — COMMENTED** [review]
Looks good, just a couple of comments.

File: tools/killsnoop.bt (hunk 3 above)
Comment (makelinux) [reply tools/killsnoop.bt:35]: The bigger problem that output of `time` and `printf` can be interleaved.

It debugging of real time issues one second is very rough.  When you have another log with hundreds of messages per second, microseconds are very helpful in finding correlations.

[strftime](https://github.com/iovisor/bpftrace/blob/master/docs/reference_guide.md#25-strftime-formatted-timestamp)  receives integer nanoseconds and provides decimal seconds only in microseconds (six digits). It can't be changed.

Comment (viktormalik) [reply tools/killsnoop.bt:35]: Ok, let's use microseconds, then.

**Review by viktormalik — APPROVED** [review]
